src/main/webapp/css/login.css

```diff
@@ -1,16 +1,11 @@
 body {
-  background: azure;
+  background: url('/img/landscape.jpg') center/cover no-repeat fixed;
```

Comment:
Remove quotes in url

https://google.github.io/styleguide/htmlcssguide.html#CSS_Quotation_Marks